src/taipy/core/config/config.schema.json

```diff
@@ -417,6 +406,14 @@
               "type": "string",
               "taipy_function": true
             }
+          },
+          "sequences": {
+            "description": "The dictionary of sequence descriptions.",
```

Comment:
is this a dictionary of a list of unique strings ?

Reply:
it's a dictionary :D

Reply:
this is the signature for the sequence in ScenarioConfig: `sequences: Optional[Dict[str, List[TaskConfig]]] = None`
It takes a str for sequence name as the key and list of TaskConfig as the value

Reply:
so how would that translate in a TOML file ?
I suppose it could be a section or an inline object ?

Reply:
I suppose for the schema they both translate into another (sequence|anonymous) Object ...

Reply:
yep it's something like this 
<img width="793" alt="image" src="https://github.com/Avaiga/taipy-core/assets/93168955/8f40416a-93ae-4aa3-bcda-1e2583b2c23f">
or we can declare it in a separate block for sequence of a scenario
<img width="441" alt="image" src="https://github.com/Avaiga/taipy-core/assets/93168955/b151bf2e-fde5-46f9-9385-7923d1724986">